### ISSUE TITLE
Các validation messange của RequestForm đang khai báo trực tiếp trong FormRequest

### ISSUE THREAD

**Issue by chungth** [hungtt57/khoaluan#6]
Không biết có lý do đặc biệt nào không ? 
A không thấy em config lang cho app, các validation message được set trực tiếp trong FormRequest như phía dưới: 

https://github.com/hungtt57/khoaluan/blob/master/app/Http/Requests/CheckThanhtoanRequest.php#L35

**Comment by hungtt57** [comment]
vâng,vì toàn dự án kiểu bài tập lớn,chưa làm khách hàng chính thống nên em config luôn
